[docs] Rephrase cheat_sheet.md (#4442)
werf/werf: docs/pages_en/reference/cheat_sheet.md

### Integrating with a CI system (GitLab and GitHub-based workflows are currently supported)

Set default values for werf commands and log in to the container registry based on GitLab environment variables:

```shell
. $(werf ci-env gitlab --as-file) 
```

### Building, tagging, and publishing images

Build images using the container registry:

```shell
werf build --repo ghcr.io/group/project
```

Build images and attach custom tags to them in addition to content-based tags:

```shell
werf build --repo ghcr.io/group/project --add-custom-tag latest --add-custom-tag 1.2.1
```

Build and store the final images in a separate registry deployed in the Kubernetes cluster:

```shell
werf build --repo ghcr.io/group/project --final-repo fast-in-cluster-registry.cluster/group/project
```

Build images using the container registry (or local storage if needed) and export them to another container registry:

```shell
werf export --repo ghcr.io/group/project --tag ghcr.io/group/otherproject/%image%:latest
```

### Running one-off tasks (unit-tests, lints, one-time jobs)

> Use the following command to run tests in the previously built `frontend_image` in the Kubernetes Pod:

Author: ```suggestion
Use the following command to run the tests in the previously built `frontend_image` in the Kubernetes Pod:
```

```shell
werf kube-run frontend_image --repo ghcr.io/group/project -- npm test
```

> Run tests in Pod, but copy file with secret env vars into container before running the command:

Author: ```suggestion
Run the tests in a Pod, but copy the file with the secret env variables to a container before executing the command:
```

```shell
werf kube-run frontend_image --repo ghcr.io/group/project --copy-to ".env:/app/.env" -- npm run e2e-tests
```

> Run tests in Pod and get the coverage report:

Author: ```suggestion
Run the tests in a Pod and get the coverage report:
```
